src/main/java/com/imperva/apiattacktool/tests/MainTest.java

```diff
@@ -81,15 +81,15 @@ private void testEndpoint(HttpRequest httpRequest, HttpResponseValidator httpRes
         try {
             httpResponse = httpClient.execute(httpHost, httpRequest);
         } catch (Exception anyException) {
-            TestReporter.log("❌ Error connecting to target! " + anyException.getMessage());
+            TestReporter.log("Error connecting to target! " + anyException.getMessage());
             TestReporter.log("");
             Assert.fail(anyException.getMessage());
             return;
         }
 
         int statusCode = httpResponse.getStatusLine().getStatusCode();
         boolean isStatusCodeValid = httpResponseValidator.isValidHttpCode(statusCode);
-        TestReporter.log((isStatusCodeValid? "✅ " : "❌ ") +"Request was: " + httpRequest.toString() + ", Response status code: " + statusCode + (isStatusCodeValid? " " : " (UNEXPECTED)"));
+        TestReporter.log((isStatusCodeValid? "ok " : "not ok ") +"Request was: " + httpRequest.toString() + ", Response status code: " + statusCode + (isStatusCodeValid? " " : " (UNEXPECTED)"));
```

Comment:
Thanks for the change. Do the visual marks create any issues? Their intention was only to help see whether there were issues visually.
I suggest having both the text and the visual marks.